posthog/api/cohort.py

```diff
@@ -190,7 +190,7 @@ def update(self, cohort: Cohort, validated_data: Dict, *args: Any, **kwargs: Any
         user = cast(User, request.user)
 
         cohort.name = validated_data.get("name", cohort.name)
-        cohort.description = validated_data.get("description", cohort.description)
+        cohort.description = validated_data.get("description", "")
```

Comment:
This looks like it breaks actual updates of the description (i.e. other than clearing it). I believe the problem is somewhere else.

Reply:
Well, it does set the description to the one in the validated data. However, I'm not in the picture if we do partial updates, so when the validated_data does not contain any description (= no change?) then we would indeed set it to blank here.
Usually one should do this more the Django way and rely on modelserializer methods anyway, like calling update 
https://github.com/encode/django-rest-framework/blob/master/rest_framework/serializers.py#L1045-L1051
They go through the fields in the validated data and only apply ones that are in there.